app/models/concerns/wakes/redirectors.rb

```diff
@@ -18,7 +18,9 @@ def create_redirect_graph
         return unless canonical_location.present?
         legacy_locations.reload
         legacy_paths_in_redis = legacy_locations.map do |legacy_location|
-          Wakes::REDIS.set(legacy_location.path, canonical_location.path_or_url) if legacy_location.host.blank?
+          if legacy_location.host.blank? || Wakes.configuration.domains.include?(legacy_location.host)
```

Comment:
@chrismaddocks I wonder if `Wakes.configuration.domains` should contain an empty string by default. Then this line could just be `if Wakes.configuration.domains.include?(legacy_location.host.to_s)` and you'd gain a (probably useless) knob to control redirecting the primary domain as well. i think it would make things more consistent.

Reply:
@benhutton I'd like to clean up this conditional, but I don't think I'm sold on the additional knob. Besides the limited utility that you point out, it feels awkward that the Wakes user will have to remember to explicitly add the empty string to this array.

Have a look at my latest commit and see how you like this approach.